Objective-C/Tests/CBLTestCase.m

```diff
@@ -131,7 +135,7 @@ - (void) reopenDB {
         Assert([_db close: &error], @"Close error: %@", error);
         _db = nil;
     }
-    [self openDB];
+    [self initDB];
```

Comment:
Should call [self openDB] otherwise if calling this in the VectorSearchTest, it will do the copy again.